_pages/en/technical-platform.md

```diff
@@ -86,7 +86,15 @@ More detailed Azure DevOps Pipeline doc: [Azure Pipeline](../assets/docs/azure-p
 
 ## Security Implications
 
-- Things we have done to make it more secure.
+- Using GraphQL mobible friendly API with strong input validation to prevent hacking the database.
+- Using containers to minimize attack surface--less code bug, less chance to be attacked. 
```

Comment:
the second half (less code bugs, less chance to be attacked) explains the benefits not so much how containers minimize attack surface (e.g. minimal OS? ports closed by default? etc.)

Reply:
i deleted the second half